### ISSUE TITLE
problem about training model on my own dataset

### ISSUE THREAD

**Issue by whrenstone** [InnerPeace-Wu/densecap-tensorflow#6]
@InnerPeace-Wu hi,  i am trying to train densecap-tensorflow project on my own dataset.But i 'm encountered a weired problem: the caption loss has been dropped from  0.4 to 0.1 by 250k iterations; but the caption  result is way out of good; as i have increased the total label class from 2 to 8, so i guess that added label classes may lead to this problem 
Could you give me some suggestions? Thanks!

**Comment by InnerPeace-Wu** [comment]
Hi, Why would you like to change to 8 classes? Firstly, you should make sure that your dataset follows the formats of VG dataset.  Besides, the code is adapted to two classes scenario specifically (refer file proposal_target_layer & proposal_target_single_class_layer). What does output look like?
Best,

**Comment by whrenstone** [comment]
for example : caption  ”学历“ for class "degree" is labeled on class "borth date" region wrongly, but the class classification is right. Besides, caption words sequence just out of order, the input caption is "电话号码" ,but the output caption is "电话性联". Hope i express the problem clear and  i can't find any code specially for two classes scenario iin proposal_target_layer...

**Comment by InnerPeace-Wu** [comment]
I don't quiet understand what's it for, but it seems you can do it just by object detection, no need for the caption. Please refer [code of single class](https://github.com/InnerPeace-Wu/densecap-tensorflow/blob/ce70133d87820f7e5d0e74ad77ae89cfae5f5b9d/lib/layers/proposal_target_single_class_layer.py#L57) and [code for multicalss](https://github.com/InnerPeace-Wu/densecap-tensorflow/blob/ce70133d87820f7e5d0e74ad77ae89cfae5f5b9d/lib/layers/proposal_target_layer.py#L51). Besides, it also affects the loss computing etc with mutliclasses.

**Comment by whrenstone** [comment]
@InnerPeace-Wu  Many thanks for your suggestion!
